packages/client/mutations/SendClientSegmentEventMutation.ts

```diff
@@ -9,14 +9,12 @@ const mutation = graphql`
   }
 `
 
-type Options = {
-  eventId?: number
-} & SegmentEventTrackOptions
+export type SendClientSegmentEventOptions = SegmentEventTrackOptions
```

Comment:
+1 I like that you've cleaned these types up. Do you think we need to assign `SegmentEventTrackOptions` to `SendClientSegmentEventOptions`? Maybe `SendClientSegmentEventOptions` is more descriptive, although as it's the same type, I'd lean towards removing this line and using `SegmentEventTrackOptions` everywhere

Reply:
I wanted to keep the related changes minimal, so I will leave it like this.